libraries/aws/shadow/include/shadow.h

```diff
@@ -0,0 +1,546 @@
+    SHADOW_TOPIC_STRING_TYPE_DELETE,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _ACCEPTED suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_DELETE_ACCEPTED,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _REJECTED suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_DELETE_REJECTED,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the
+     * one with suffix. This rule is suppressed for naming consistency with
+     * other Shadow header field and value string and length macros in this file.*/
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_UPDATE,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _ACCEPTED suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_UPDATE_ACCEPTED,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _REJECTED suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_UPDATE_REJECTED,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _DOCUMENTS suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_UPDATE_DOCUMENTS,
+
+    /* MISRA Rule 5.2 flags the following member's name as ambiguous from the one
+     * without _DELTA suffix. This rule is suppressed for naming consistency with other Shadow
+     * header field and value string and length macros in this file.*/
+    /* coverity[misra_c_2012_rule_5_2_violation] */
+    /* coverity[other_declaration] */
+    SHADOW_TOPIC_STRING_TYPE_UPDATE_DELTA,
+    SHADOW_TOPIC_STRING_TYPE_MAX_NUM
+} ShadowTopicStringType_t;
+
+/**
+ * @brief Return codes from Shadow functions.
+ */
+typedef enum ShadowStatus
+{
+    SHADOW_STATUS_SUCCESS = 0,                          /**< @brief Shadow function success. */
```

Comment:
Just checked and upper snake case seems consistent with what HTTP uses for statuses, and MQTT uses upper camel case. I'd at least take out the `STATUS` from each of these names, but otherwise this change isn't necessary right now and can be revisited later when the libraries are to be released

Reply:
will remove "STATUS" in next commit

Reply:
fix it in commit [a1d235a1](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1081/commits/a1d235a16c797862dc016f47bbf0e1437f2efaba)

resolving this comment

feel free to let me know if it needs further changes, thank you